test/data.sql

```diff
@@ -2930,10 +2930,10 @@ INSERT INTO `voucher_item` (`uuid`, `account_id`, `debit`, `credit`, `voucher_uu
   (HUID('90583c32-b1b2-11e8-9689-0b54421d0e49'), 187, 100, 0, @first_voucher, @first_invoice, HUID('2c6c48a2-b1b3-11e8-ae9b-1fa4024347ab')),
   (HUID('9317a11a-b1b2-11e8-93d6-b30828591803'), 182, 0, 100, @first_voucher, NULL, NULL),
   (HUID('941ae478-b1b2-11e8-9492-6385e74c37a0'), 188, 200, 0, @second_voucher, NULL, NULL),
-  (HUID('953defee-b1b2-11e8-ac16-3b802412d42d'), 200, 0, 200, @second_voucher, NULL, NULL),
+  (HUID('953defee-b1b2-11e8-ac16-3b802412d42d'), 242, 0, 200, @second_voucher, NULL, NULL),
   (HUID('97ad9a22-b1b2-11e8-acba-6b85fd27d57d'), 125, 300, 0, @third_voucher, @cash_payment, HUID('2f0b966c-b1b3-11e8-8dd2-1715a827ad9b')),
   (HUID('8ba0571a-b1b2-11e8-8688-fb9547361273'), 117, 0, 300, @third_voucher, NULL, NULL),
```

Comment:
On this line, the account identifier 117 corresponds to that of the account [165 - Deposits and  Guarantees received]